include/verilated_saif_c.h

```diff
@@ -0,0 +1,430 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//=============================================================================
+//
+// Code available from: https://verilator.org
+//
+// Copyright 2001-2025 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//=============================================================================
+///
+/// \file
+/// \brief Verilated tracing in SAIF format header
+///
+/// User wrapper code should use this header when creating SAIF traces.
+///
+//=============================================================================
+
+#ifndef VERILATOR_VERILATED_SAIF_C_H_
+#define VERILATOR_VERILATED_SAIF_C_H_
+
+#include "verilated.h"
+#include "verilated_trace.h"
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+class VerilatedSaifBuffer;
+class VerilatedSaifFile;
+
+//=============================================================================
+// ActivityBit
+
+class ActivityBit final {
```

Comment:
```suggestion
class VerilatedSaifActivityBit final {
```

& similar all classes. Need to avoid collisions as can compile all .cpp files in large project as one.